plugins/eos/gs-plugin-eos.c

```diff
@@ -39,6 +39,7 @@
 #define EOS_IMAGE_VERSION_ALT_PATH "/"
 
 #define METADATA_SYS_DESKTOP_FILE "EndlessOS::system-desktop-file"
+#define METADATA_REPLACED_BY_DESKTOP_FILE "Endless::ReplacedByDesktopFile"
```

Comment:
Please make both the namespace and the capitalization style of this key consistent with the other custom metadata key(s) we already have (e.g. the one defined just above)